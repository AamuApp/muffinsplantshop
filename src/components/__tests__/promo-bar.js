import React from "react";
import { shallow } from 'enzyme';

import PromoBar from '../promo-bar/promo-bar';

test('Should render PromoBar correctly', () => {
  const wrapper = shallow(<PromoBar />);
  expect(wrapper).toMatchSnapshot();
});