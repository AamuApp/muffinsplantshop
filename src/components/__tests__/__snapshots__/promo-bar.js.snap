// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render PromoBar correctly 1`] = `
<div
  className="promobar"
>
  <p
    className="promobar__text"
  >
    Free delivery in Downtown Ottawa
  </p>
</div>
`;
